source/_integrations/switchbot_cloud.markdown

```diff
@@ -32,11 +34,18 @@ Please note, device names configured in the SwitchBot app are transferred into H
 
 - Plug (Wi-Fi only, only available in Japan)
 - Plug Mini, both the original and HomeKit-enabled
-
 - IR appliances exposed through the different hubs:
   - ON/OFF for all appliance types excepted "Others"
   - Air Conditioner
+- Sensors (Meter, MeterPlus, Outdoor Meter)
+
+## Important considerations
+
+By default, each individual sensor will poll the SwitchBot Cloud API for a status update every 10 minutes (600 seconds). You can request more frequent updates by lowering the `Scan interval (seconds)` in the initial integration setup.
 
+<div class='note warning'>
+The SwitchBot Cloud API limits users to 10,000 requests per day. Please be cautious when lowering the scan interval.
+</div>
```

Comment:
This needs rewording

Reply:
Have reworded this. I figure it's good to just give users the information about the request limit. They can do with that what they wish.